spec/concepts/todo_list/create_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+RSpec.describe TodoList::Create, type: :operation do
+  it "creates a default todo list" do
+    todo_list = TodoList::Create.(todo_list:
+      { id: nil, user_id: nil}
+    ).model
+
+    expect(todo_list.name).to eq("Default To-do List")
```

Comment:
It will create another to-do list with the name you provided if it's not yet created? :)